myhpi/templates/base.html

```diff
@@ -34,25 +34,53 @@
 
 <body class="bg-background myhpi-accent-yellow {% block body_class %}{% endblock %}">
     {% wagtailuserbar %}
-
-    <nav class="navbar-myhpi container-lg page d-print-none">
-        <div class="navbar-top bg-white">
-            {% with all_pages|all_sub_menu_pages_by_id as sub_pages_by_level_and_id %}
-            <div class="page-content d-flex">
-                <a class="nav-brand" href="{% pageurl root_page %}">
-                    <img src="{% static 'img/FSR_DE_color.svg' %}" class="w-100" alt="Zur Startseite">
-                </a>
-                {% build_menu_level sub_pages_by_level_and_id 0 %}
+    <nav class="navbar-myhpi container-lg d-print-none page">
+        <div class="bg-white">
+            <div class="navbar-top d-flex flex-between page-content">
+                {% with all_pages|all_sub_menu_pages_by_id as sub_pages_by_level_and_id %}
+                <div class="d-flex flex-between">
+                    <a class="nav-brand" href="{% pageurl root_page %}">
+                        <img src="{% static 'img/FSR_DE_color.svg' %}" class="w-100" alt="Zur Startseite">
```

Comment:
```suggestion
                        <img src="{% static 'img/FSR_DE_color.svg' %}" class="w-100" alt="To home page">
```

Reply:
we could already use the translate template tags here. needs to be adapted soon anyways

Reply:
Will wagtail provide the details of the translated strings automatically (via settings or similar), or do we need to implement that ourselves?

Reply:
the translate tag is a built-in function of django, see https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#translate-template-tag.
The translations are stored in a specific file in the repository and django takes care of displaying the correct string for the current language

Reply:
Is there a particular style you wish for me to follow or is any style I deem fit fine?